File: PF with Python Jan 2022/15. Functions - Exercise/01. smallest_of_three_numbers.py
# 1.Smallest of Three Numbers
# Write a function that receives three integer numbers and returns the smallest. Print the result on the console. Use an appropriate name for the function.
#
# Input
# 2
# 5
# 3
#
# Output
# 2

def smallest(a, b, c):
    return min(a, b, c)

num1, num2, num3 = int(input()), int(input()), int(input())
result = smallest(num1, num2, num3)

print(result)

File: JS Front-End/06. Exercise Functions and Statements/01. smallest-of-three-numbers.js
function smallestOfThreeInt(...numbers) {
  return Math.min(...numbers);
}

console.log(smallestOfThreeInt(2, 5, 3));
console.log(smallestOfThreeInt(600, 342, 123));
console.log(smallestOfThreeInt(25, 21, 4));
console.log(smallestOfThreeInt(2, -2, 2));
console.log(smallestOfThreeInt(2, -2, 2.5, 4, -8, -8.9));
